docker/secondary-analysis-python/tools/submit/submit/get_staging_urn.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import requests
+import json
+import argparse
+import time
+
+def run(envelope_url, retry_seconds, timeout_seconds):
+    start = time.time()
+    current = start
+    urn = None
+    while True:
+        envelope_js = get_envelope_json(envelope_url)
+        urn = get_staging_urn(envelope_js)
+        if urn:
+            break
+        time.sleep(retry_seconds)
+        current = time.time()
+        if current - start >= timeout_seconds:
+            message = 'Timed out while trying to get urn. Timeout seconds: {}'.format(timeout_seconds)
+            raise ValueError(message)
+    print(urn)
+
+def get_envelope_json(envelope_url):
+    response = requests.get(envelope_url)
+    envelope_js = response.json()
+    return envelope_js
+
+def get_staging_urn(envelope_js):
+    details = envelope_js.get('stagingDetails')
+    if not details:
+        return None
+    location = details.get('stagingAreaLocation')
+    if not location:
+        return None
+    urn = location.get('value')
+    if not urn:
```

Comment:
You can remove this `if` statement, since `return urn` would just return `urn` or `None`

Reply:
Good point!